.prettierignore

```diff
@@ -0,0 +1,12 @@
+.git/
+coverage
+assets/
+package-lock.json
+package.json
+
+public/css/
+public/js
+public/vendor/
+build/
+**/*.bundle.js
+**/*.md
```

Comment:
Should this also include `**/*.css`?

The components' less files get transformed to CSS, but we don't want to prettify those.